algo/src/main/java/org/neo4j/graphalgo/impl/HugePageRank.java

```diff
@@ -209,6 +205,7 @@ private void initializeSteps() {
                 concurrency,
                 idMapping.nodeCount(),
                 dampingFactor,
+                sourceNodeIds.map(graph::toMappedNodeId).collect(Collectors.toList()),
```

Comment:
Since this is `Huge`, we should use `toHugeMappedNodeId` to get the internal long id.

Reply:
Also, a Node may not have been mapped if there was any Label/Type filtering, so maybe add a `filter(mappedId -> mappedId != -1L)` to remove the nodes that we don't know about.